docs/source/en/tasks/image_classification.mdx

```diff
@@ -155,21 +194,23 @@ Then create a function that passes your predictions and labels to [`~evaluate.Ev
 
 
 >>> def compute_metrics(eval_pred):
-...     predictions = np.argmax(eval_pred.predictions, axis=1)
-...     return accuracy.compute(predictions=predictions, references=eval_pred.label_ids)
+...     predictions, labels = eval_pred
+...     predictions = np.argmax(predictions, axis=1)
+...     return accuracy.compute(predictions=predictions, references=labels)
 ```
 
-Your `compute_metrics` function is ready to go now, and you'll return to it when you setup your training.
+Your `compute_metrics` function is ready to go now, and you'll return to it when you set up your training.
 
 ## Train
 
 <frameworkcontent>
 <pt>
 <Tip>
 
-If you aren't familiar with finetuning a model with the [`Trainer`], take a look at the basic tutorial [here](../training#train-with-pytorch-trainer)!
+If you aren't familiar with fine-tuning a model with the [`Trainer`], take a look at the basic tutorial [here](../training#train-with-pytorch-trainer)!
```

Comment:
I have said it before, there is no standard for finetune vs fine-tune in the library and we won't enforce one. While you're free to pick the one you like when writing new content, please leave the existing ones as is so that the diff is easier to read for the reviewer :-)